web/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM nginx:alpine
+
+LABEL varion="1.0" \
+    maintainer="h1dden-da3m0n" \
+    description="A simple TeaSpeak WebClinet running on Nginx-Alpine"
+
+ARG WEB_VERSION="9a23f96"
+
+ENV WEB_VERSION="${WEB_VERSION}" \
+    TZ="Europe/Berlin"
+
+COPY ./default.conf /etc/nginx/conf.d/default.conf
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./entrypoint.sh /
+
+RUN apk update --no-cache && apk upgrade --no-cache \
+    && apk add --no-cache openssl tzdata \
+    \
+    && mkdir -p /var/www/TeaWeb /etc/ssl/certs \
+    && wget -nv -O /var/www/TeaWeb/TeaWeb-release.zip \
+        https://github.com/TeaSpeak/TeaWeb/releases/download/${WEB_VERSION}/TeaWeb-release.zip \
+    && unzip -oq /var/www/TeaWeb/TeaWeb-release.zip -d /var/www/TeaWeb \
+    && rm //var/www/TeaWeb/TeaWeb-release.zip \
+    && sed -i /etc/nginx/mime.types \
+        -e 's/}/    application\/wasm    wasm;\n}/' \
+    && chmod +x /entrypoint.sh
+
+VOLUME ["/etc/ssl/certs"]
```

Comment:
No the web client is only client sided. It does not communicate with the server at all, e.g. stores data.
The server is just there to provide the required files. Since you're not even requiring PHP anymore, it might also work locally without the need of a web server at all. Would be a fun thing to test tbh.

Reply:
I think you may have misunderstood my question, but I think elaborating that and figuring out if it was the case will be best over voice (eg. later in TS)